src/components/Login.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+<div class="hero">
+  <div class="container h-100">
+      <div class="row h-100 align-items-center justify-content-center text-center">
+        <div class="col-lg-10 align-center">
+          <h1 class="title">SuperDel</h1>
+          <hr class="divider my-4">
+          <p class="sub-title">Grocery shopping made easy and accessible for all!</p>
+        </div>
+        <LoginUser />
+      </div> 
+  </div>
+</div>
+</template>
+
+<script>
+import LoginUser from './LoginUser'
+
+export default {
+  name: 'Login',
+  components: {
+    LoginUser
+  },
+  props: {
```

Comment:
Store local things in data. Because we are using Vuex, we won't have a need for props.